lib/etl_cases.rb

```diff
@@ -0,0 +1,54 @@
+require 'exercise_cases'
+
+class EtlCase < OpenStruct
+  def test_name
+    'test_%s' % description.tr(' ,-', '_').downcase
+  end
+
+  def workload
+    [
+      "old = #{format_hash(integerize_keys(input))}",
+      "    expected = #{format_hash(expected)}\n",
+      indent(4, assertion),
+    ].join("\n")
+  end
+
+  def skipped
+    index.zero? ? '# skip' : 'skip'
+  end
+
+  private
+
+  def indent(size, lines)
+    lines.lines.each_with_object('') { |line, obj| obj << ' ' * size + line }
+  end
+
+  def assertion
+    "assert_equal expected, ETL.transform(old)"
+  end
+
+  def integerize_keys(input)
+    input.reduce({}) { |hash, (k, v)| hash[k.to_i] = v; hash }
+  end
+
+  def format_hash(hash)
```

Comment:
I think this method can be simplified a bit. 

Try operating on the hash as a hash, rather than as a string.